README.md

```diff
@@ -30,16 +37,22 @@ $ dbt seed
 
 ### Running Instructions:
 
-Run fal flow
+1. Run `dbt run`
```

Comment:
This part reads like the user should do both `dbt run` and `fal flow`. Is that the case or should the user choose one or the other?

Reply:
Agreed